### ISSUE TITLE
Implement: Student Interest Drilldown Page

### ISSUE THREAD

**Issue by philipmjohnson** [radgrad/radgrad2#409]
Please do work for this task in a branch called issue-409.

## Current Page Design

<img width="1451" alt="Screen Shot 2021-02-16 at 1 53 14 PM" src="https://user-images.githubusercontent.com/290288/108136580-6cd3f900-705e-11eb-9f53-e41c1a5c7f21.png">

## UI Issues

* The page is not yet converted to use `<PageLayout>` and `<HeaderPane>`
* Get rid of the drop down menu to go to other explorers. That's available n the navbar.
* Get rid of the "Suggest a new interest" button. The user should be doing that from the page that lists all of the interests, to ensure they don't suggest an interest that already exists.
* If the related courses and opportunities were laid out horizontally, then there wouldn't be the vertical gutters of white space, and then the entire page would fit without scrolling
* 65 Students without photos looks weird. And they aren't clickable.
* Faculty are not clickable.
* Use name and add image only if available?
* Add to Favorites -> Add to Profile

## UX Issues
* How do we communicate that this page is where you manage your profile interests?
* How do we build community from this page effectively?

**Comment by philipmjohnson** [comment]
Mockup redesign is complete:
https://www.figma.com/file/n96EJt61wuPnR14YxkFS4m/RadGrad2-interests?node-id=0%3A1

Now can move to implementation phase:

<img width="900" alt="Screen Shot 2021-03-01 at 10 47 55 AM" src="https://user-images.githubusercontent.com/290288/109557066-af90ca80-7a7b-11eb-9bb3-620282f50a72.png">

**Comment by cammoore** [comment]
I've made a few changes to the mockup.
![Uploading Screen Shot 2021-03-17 at 10.57.59 AM.png…]()

**Comment by cammoore** [comment]
Merged branch into master. Going to wait on the profile labels.

**Comment by philipmjohnson** [comment]
## Mockup revision

Note we will eliminate the text from the HeaderPanel. Maybe move the button to float left? 

<img width="1005" alt="Screen Shot 2021-03-22 at 11 27 54 AM" src="https://user-images.githubusercontent.com/290288/112060885-d897fe80-8b01-11eb-87cd-cabc7f5f5d24.png">

**Comment by cammoore** [comment]
Closing. We need another issue to add an image for Interests.
